pkg/scheduler/internal/channels/error_channel.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package channels
```

Comment:
`channels` may not be a good name for package, `channel` would be better as a package name here.

Reply:
@draveness @alculquicondor What package name would your suggest to go with?

Reply:
I prefer `errchannel`.New` to `channel.NewErrorChannel`

Reply:
being more specific, as @draveness suggested, reads well too.